brew install lastpass-cli --with-pinentry fails

### Bug reports:
I am running brew v1.1.9 on OSX 10.12.3

I ran `brew install lastpass-cli --with-pinentry`

and received 
```bash
==> Using the sandbox
==> Downloading https://github.com/lastpass/lastpass-cli/archive/v1.1.2.tar.gz
Already downloaded: /Users/lphilbrook/Library/Caches/Homebrew/lastpass-cli-1.1.2.tar.gz
==> make PREFIX=/usr/local/Cellar/lastpass-cli/1.1.2 install
==> make MANDIR=/usr/local/Cellar/lastpass-cli/1.1.2/share/man install-doc
Last 15 lines from /Users/lphilbrook/Library/Logs/Homebrew/lastpass-cli/02.make:
to the PKG_CONFIG_PATH environment variable
No package 'bash-completion' found
-- Configuring done
-- Generating done
-- Build files have been written to: /tmp/lastpass-cli-20170207-12595-my48hu/lastpass-cli-1.1.2/build
/Library/Developer/CommandLineTools/usr/bin/make -C build install-doc
Scanning dependencies of target doc-man
[100%] Generating lpass.1
a2x: WARNING: --destination-dir option is only applicable to HTML based outputs
a2x: ERROR: "xsltproc"  --stringparam callout.graphics 0 --stringparam navig.graphics 0 --stringparam admon.textlabel 1 --stringparam admon.graphics 0  "/usr/local/Cellar/asciidoc/8.6.9_1/etc/asciidoc/docbook-xsl/manpage.xsl" "/private/tmp/lastpass-cli-20170207-12595-my48hu/lastpass-cli-1.1.2/build/lpass.1.xml" returned non-zero exit status 5
make[4]: *** [lpass.1] Error 1
make[3]: *** [CMakeFiles/doc-man.dir/all] Error 2
make[2]: *** [CMakeFiles/install-doc.dir/rule] Error 2
make[1]: *** [install-doc] Error 2
make: *** [install-doc] Error 2

READ THIS: http://docs.brew.sh/Troubleshooting.html
```

Running these commands first seemed to solve it.
```bash
brew reinstall asciidoc --with-docbook-xsl
export XML_CATALOG_FILES=/usr/local/etc/xml/catalog
```

The only way I can persistently reproduce this is my denying the process access to the internet.
```
[100%] Generating lpass.1
a2x: WARNING: --destination-dir option is only applicable to HTML based outputs
a2x: ERROR: "xsltproc"  --stringparam callout.graphics 0 --stringparam navig.graphics 0 --stringparam admon.textlabel 1 --stringparam admon.graphics 0  "/usr/local/Cellar/asciidoc/8.6.9_1/etc/asciidoc/docbook-xsl/manpage.xsl" "/private/tmp/lastpass-cli-20171020-76144-jqkrae/lastpass-cli-1.2.1/build/lpass.1.xml" returned non-zero exit status 5
make[4]: *** [lpass.1] Error 1
make[3]: *** [CMakeFiles/doc-man.dir/all] Error 2
make[2]: *** [CMakeFiles/install-doc.dir/rule] Error 2
make[1]: *** [install-doc] Error 2
make: *** [install-doc] Error 2
```
Otherwise 🤷‍♂️.

--- a/Formula/lastpass-cli.rb
+++ b/Formula/lastpass-cli.rb
@@ -15,11 +15,13 @@ class LastpassCli < Formula
 
   depends_on "asciidoc" => :build
   depends_on "cmake" => :build
+  depends_on "docbook-xsl" => :build
   depends_on "pkg-config" => :build
   depends_on "openssl"
   depends_on "pinentry" => :optional
 
   def install
+    ENV["XML_CATALOG_FILES"] = etc/"xml/catalog"
     system "make", "PREFIX=#{prefix}", "install"
     system "make", "MANDIR=#{man}", "install-doc"
   end

